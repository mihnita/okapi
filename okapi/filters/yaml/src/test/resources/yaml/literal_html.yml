dialog: |
  <p>Some literal HTML text that should be
    processed as a block</p>
  <ul>
    <li>item 1</li>
    <li>item 2</li>
    <li>item 3</li>
  </ul>
not: not literal
