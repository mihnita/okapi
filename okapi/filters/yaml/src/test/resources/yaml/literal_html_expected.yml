dialog: |
  <p>SOME LITERAL HTML TEXT THAT SHOULD BE PROCESSED AS A BLOCK</p> <ul> <li>ITEM 1</li> <li>ITEM 2</li> <li>ITEM 3</li> </ul>
not: NOT LITERAL
